texasbbq.py

```diff
@@ -41,10 +41,12 @@
 
 
 def echo(value):
+    """ Print to stdout with prefix. """
```

Comment:
Why do all these docstrings start and end with spaces?

Reply:
I used this style in a different project, let me use `black` here.

Reply:
I fixed the issue and added more missing docstrings